apps/marsha/templates/services/app/cronjob_check_harvested.yml.j2

```diff
@@ -0,0 +1,69 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  labels:
+    app: marsha
+    service: app
+    version: "{{ marsha_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "check-harvested-{{ deployment_stamp }}"
+  namespace: "{{ project_name }}"
+spec:
+  schedule: "0 3 * * *"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Forbid
+  suspend: {{ suspend_cronjob | default(false) }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: "check-harvested-{{ deployment_stamp }}"
+          labels:
+            app: marsha
+            service: app
+            version: "{{ marsha_image_tag }}"
```

Comment:
I think we stopped putting this label?

Reply:
Ok I didn't know. But we are still using them in marsha at least.

Reply:
that's part of @madmatah's recent work.

Reply:
The label `version` has been removed on `Deployment`s (in `spec.selector` section ) and in `Static Services`.
Its use has not changed for the rest of the k8s entities.